src/main/java/org/jenkinsci/lib/envinject/service/EnvInjectActionRetriever.java

```diff
@@ -16,15 +19,28 @@
     //with EnvInjectAction subclasses. Subclasses cannot be casted from
     //all point of Jenkins (classes are not loaded in some points)
     public Action getEnvInjectAction(AbstractBuild<?, ?> build) {
-
+        List<Action> actions;
```

Comment:
You could keep this bellow where it was. If build is null you don't need this variable.